metadata.yaml

```diff
@@ -21,10 +21,8 @@ peers:
 provides:
   zookeeper:
     interface: zookeeper
-  metrics-endpoint:
-    interface: prometheus_scrape
-  grafana-dashboard:
-    interface: grafana_dashboard
+  cos-agent:
+    interface: cos_agent
```

Comment:
**suggestion:** Limit subord. rel. count to 1.
```suggestion
    interface: cos_agent
    limit: 1
```
As @rbarry82 rightfully [points out](https://github.com/canonical/grafana-agent-k8s-operator/issues/144), it is unlikely for an admin to relate two different grafana-agent subords to the same principal, and is considered a "don't do that" in the juju rule book. However, it would be nice to explicitly restrict it.